cdk/lib/gatehouse.ts

```diff
@@ -34,7 +34,7 @@ export class Gatehouse extends GuStack {
             userData: {
                 distributable: {
                     fileName: `${ec2App}.deb`,
-                    executionStatement: `dpkg -i /${ec2App}/${ec2App}.deb && echo "stage=${this.stage}" | sudo tee "/etc/gatehouse/stage.conf" > /dev/null`,
+                    executionStatement: `dpkg -i /${ec2App}/${ec2App}.deb`,
```

Comment:
Because we're getting the stage from EC2 tags, we no longer need to write a file with the stage value in it.